cmd/commandfuncs.go

```diff
@@ -40,6 +41,8 @@ import (
 	"go.uber.org/zap"
 )
 
+const DownloadPath = "https://caddyserver.com/api/download"
+
```

Comment:
We tend to prefer putting consts at the bottom of the file. See `/caddy.go` for example, `var` and `const` at the end.